Core/src/Seeding2/TripletSeeder.cpp

```diff
@@ -26,16 +26,17 @@ void createAndFilterTriplets(TripletSeeder::Cache& cache,
                              DoubletCollections bottomDoublets,
                              const ConstSpacePointProxy2& spM,
                              DoubletCollections topDoublets) {
-  for (DoubletsForMiddleSp::Proxy lb : bottomDoublets) {
+  for (auto bottomDoublet : bottomDoublets) {
```

Comment:
`bottomDoublet` is a `Proxy`? I think Sonar will recommend using `const auto&` anyway, which I think should be safe from a lifetime point of view.

Reply:
it can be either a `Proxy` or a `Proxy2`